packages/sit-onyx/src/i18n/locales/de-DE.json

```diff
@@ -27,9 +27,13 @@
     "searchInputLabel": "Liste durchsuchen",
     "clearSearch": "Suchtext löschen"
   },
+  "link": {
+    "opensExternally": "Wird in einem neuen Tab geöffnet."
+  },
   "navItemOptionsLabel": "Unterseiten von {label}",
   "navigation": {
-    "appLogo": "App logo",
+    "appLogo": "App logo von {appName}",
+    "goToHome": "Gehe zur Startseite",
```

Comment:
Technically the app area click is up to the project so it does not have to go to the home page

Reply:
Yes, that's why the label is configurable. But we could also make the label required, if you prefer